FrontEnd/src/components/profileList/ProfileCard.module.css

```diff
@@ -107,7 +107,7 @@
 }
 
 .content__common-info {
-  width: 1169px;
+  width: 1150px;
```

Comment:
Did you test whether it works with screens different than 1150?

Reply:
Everything seems to work ok with this task on various resolutions, but this whole issue with hardcoded sizes in pixels will be fixed in future